### DISCUSSION TITLE
Features request - user-colour, user-transition-time.

### DISCUSSION THREAD

**Discussion by clearoak** [Tyrrrz/LightBulb#245] [General]
Thanks for your work on this.   I hope you're not too near the oil/gas fields that Russia wants.   
I would like to mention two features that I'd like:
I'd like the night colour to be user-selected.    At the moment, I find it not red enough. 
Also I would like the time duration of the sunset transition to be user-selected. 
Otherwise, I appreciate the manual selection of sunrise and sunset, and is the reason I switched to this program. 
(Windows.)

**Comment by Tyrrrz** [comment]
> I'd like the night colour to be user-selected. At the moment, I find it not red enough.
> Also I would like the time duration of the sunset transition to be user-selected.

You should already be able to do that. The night color = night temperature. You can tweak the numbers lower by typing them manually instead of using the slider. As for time duration, check out "transition duration" and "[transition offset](https://github.com/Tyrrrz/LightBulb/wiki/How-the-transition-offset-works)".
